docs/decisions/0005-toggle-life-expectancy.rst

```diff
@@ -0,0 +1,92 @@
+
+#. Add ``toggle_life_expectancy`` with values of "temporary" or "permanent".
+#. Make ``toggle_use_cases`` required when ``toggle_life_expectancy`` was "permanent".
+
+The second point was proposed to force people to be more intentional around "permanent" toggles, so someone could not simply mark a toggle as "permanent" to avoid the clean-up required for "temporary" toggles without hopefully being a bit more intentional.
+
+In the `PR to implement this proposal`_, the solution was tweaked based on some lost decisions. The update adjusted the ``toggle_uses_cases`` to the following values:
+
+* *temporary*: A new use case that replaced all uses case options that were considered temporary.
+* *circuit_breaker*: Replaced "graceful_degradation", although the OEP has not yet been updated.
+* *vip, opt_out, opt_in, open_edx*: The other original permanent use case options.
+
+The current implementation has the following flaws:
+
+* Defining "permanent" as the absence of "temporary" reduces clarity for "permanent" toggles.
+* The usage of use cases isn't clear.
+
+  * The options "circuit_breaker" and "opt_out" ("opt_in") seem like two types of toggles. The former is ops related, and the latter is business related.
+
+    * Note that "opt_out" and "opt_in" only differ in language based on the default of the toggle. For example, if the default is True, then "opt_out" linguistically makes sense as the purpose of the toggle, and if the default is False, "opt_in" makes more sense.
+
+  * The options "vip" vs "open_edx" seem more about the scope at which one might apply the ability to opt-out/opt-in. However, this scope may vary depending on the Open edX instance, and it may not make sense to document the scope selected for edX.org. Additionally, the ``toggle_implementation`` annotation may duplicate this information, assuming the scoping features of the implementation match the intent.
+  * In practice, it seems that "open_edx" is often chosen as a general option for permanent toggles, and negates the intent to use ``toggle_use_cases`` to force intentional design and consideration for permanent toggles.
+
+.. _set of use cases for toggles: https://github.com/edx/open-edx-proposals/blob/c2d3b2a/oeps/oep-0017-bp-feature-toggles.rst#use-cases
+.. _Details a reporting requirement: https://github.com/edx/open-edx-proposals/blob/c2d3b2a/oeps/oep-0017-bp-feature-toggles.rst#req-12-report
+.. _PR discussion about toggle_use_cases: https://github.com/edx/edx-platform/pull/24815#issuecomment-681174018
+.. _PR to implement this proposal: https://github.com/edx/code-annotations/pull/49
+
+Decision
+--------
+
+In order to make the toggle life expectancy more clear, and the justification
+for permanent toggles more clear, we will take the following actions:
+
+
+* As originally proposed, introduce ``toggle_life_expectancy`` with values of "temporary" or "permanent". This makes the options very clear.
```

Comment:
So now `toggle_life_expectancy: temporary` is what will obligate a `toggle_target_removal_date` annotation?

Reply:
Added a note about this. Thanks!